### DISCUSSION TITLE
18.19.1?

### DISCUSSION THREAD

**Discussion by michaeldice** [nodejs/Release#994] [Q&A]
Do you have a timeframe when 18.19.1 will be available under the Linux (or Windows) app service plans?

**Comment by richardlau** [comment]
What's an app service plan? (You're probably asking this question to the wrong people.)

**Comment by SoeAung95** [comment]
pnpm env use --global nightly
pnpm env use --global rc
pnpm env use --global 16.0.0-rc.0
pnpm env use --global rc/14

**Reply by jqywmgycdn-sys** [reply]
Done

**Reply by jqywmgycdn-sys** [reply]
- [ ] - > pnpm env use --global nightly
> 
> pnpm env use --global rc
> 
> pnpm env use --global 16.0.0-rc.0
> 
> pnpm env use --global rc/14

> pnpm env use --global nightly
> 
> pnpm env use --global rc
> 
> pnpm env use --global 16.0.0-rc.0
> 
> pnpm env use --global rc/14
